tronpy/providers/http.py

```diff
@@ -56,6 +58,11 @@ def __init__(
             self._default_api_keys = self._api_keys.copy()
         else:
             self.use_api_key = False
+        
+        if jw_token is None or isinstance(jw_token, str):
+            self.jw_token = f"Bearer {jw_token}"
+        else:
+            raise TypeError("unknown jwt type {}".format(jw_token))
```

Comment:
In general I think though that something like

self.jw_token = jw_token

...

if self.jw_token:
     self.sess.headers["Authorization"] = f"Bearer {jw_token}"

Reply:
i was thinking to follow the existing code format by adding the headers in the `make_request` method